run

```diff
@@ -155,12 +155,45 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
             shift
         ;;
         -h|--help) echo "$USAGE"; exit ;;
-        -v|--version) echo "Generated from generator-canonical-webteam@3.2.1"; exit ;;
+        -v|--version) echo "Generated from generator-canonical-webteam@3.3.1"; exit ;;
         *) invalid "Option '${key}' not recognised." ;;
     esac
     shift
 done
 
+start_django_db () {
+    # Run the database if necessary
+    if grep -q django.db.backends.postgresql_psycopg2 */settings.py 2> /dev/null; then
+        # Create isolated network
+        if ! docker network inspect ${network_name} &> /dev/null; then
+            docker network create ${network_name}
+        fi
+
+        # Start the database
+        trap "kill_container ${db_container}" EXIT;
+        if ! docker inspect -f {{.State.Running}} ${db_container} &>/dev/null; then
+            docker run \
+                --name ${db_container}       `# Name the container`  \
+                --rm                         `# Remove the container once it's finished`  \
+                --volume "${db_volume}":/var/lib/postgresql/data  `# Store dependencies in a docker volume`  \
```

Comment:
Saves PG data, not dependencies.

Reply:
Mentioned in https://github.com/canonical-webteam/generator-canonical-webteam/issues/105